caddyhttp/errors/errors.go

```diff
@@ -50,7 +49,7 @@ func (h ErrorHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, er
 	status, err := h.Next.ServeHTTP(w, r)
 
 	if err != nil {
-		errMsg := fmt.Sprintf("%s [ERROR %d %s] %v", time.Now().Format(timeFormat), status, r.URL.Path, err)
+		errMsg := fmt.Sprintf("[ERROR %d %s] %v", status, r.URL.Path, err)
```

Comment:
@mholt Prepeding a timestamp should be up to the `httpserver.Logger` that prints a line.
Should that logger be adjusted?

Reply:
Yes, I suppose so